peer_mentoring/chat/views.py

```diff
@@ -20,3 +21,26 @@ def group_chat_room(request, group_id):
         "chat_messages": chat_messages,
     }
     return render(request, "chat/room.html", context)
+
+
+@login_required
+def private_chat_room(request, sender_id, receiver_id):
+    sender = User.objects.get(pk=sender_id)
+    receiver_id = User.objects.get(id=receiver_id)
+    private_chat_messages = PrivateChatMessage.objects.filter(
+        sender=sender, receiver=receiver_id
+    )
+
+    private_group = sender.friends.filter(receiver=receiver_id)
+    if private_group.count() == 0:
+        return HttpResponseForbidden()
```

Comment:
not sure user experience, could also use messages.error(request, ... then redirect to home